packages/synthetix-main/deployments/kovan/official/2021-11-17-00.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I had two failed attempts because of the race condition fix pointed out above. It's nice to see how the deployer kept a record of everything.